# Description of the stop line on the map

uint64  id
float64 s